mobile/src/main/java/rs/readahead/washington/mobile/views/base_ui/BaseFragment.kt

```diff
@@ -12,11 +12,17 @@ import androidx.navigation.fragment.NavHostFragment
 import timber.log.Timber
 import com.tooltip.Tooltip
 import rs.readahead.washington.mobile.R
+import rs.readahead.washington.mobile.util.setupForAccessibility
 
 abstract class BaseFragment : Fragment() {
 
     protected lateinit var baseActivity: BaseActivity
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        childFragmentManager.setupForAccessibility()
+    }
```

Comment:
The listener is called whenever there's a change in the back stack, and iterating through fragments and setting their accessibility properties could be relatively expensive if done frequently. -> this could lead to a bad performance what do you think @wafalakhal ?